config/config.sample.yaml

```diff
@@ -112,4 +112,8 @@ provisioning:
 
 puppeting:
   # Should the bridge allow users to puppet their accounts
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+bot_profile: # Set the bot's profile on startup, optional
+  displayname: Slack Bridge # Set the displayname, optional
+  avatar_url: "mxc://half-shot.uk/ea64c71ee946ca2f61379abefe2c7d977d276fbb" # Set the avatar, optional
```

Comment:
I'd comment out all of `bot_profile` as it's optional.
Do you want to keep your MXC URL in `avatar_url`?

Reply:
Hmm, so I kept it in because in terms of defaults, I like the idea of people's bots having profiles when they set the bridge up. I left it as optional, because there will be cases where people don't want this, but I think it should be opt out.